build.gradle

```diff
@@ -21,14 +22,15 @@ repositories {
         url "https://nexus.gluonhq.com/nexus/content/repositories/releases"
     }
     maven {
-        url "https://nexus.gluonhq.com/nexus/content/repositories/public-snapshots"
+        url "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
You may be able to pull substrate snapshots from your Nexus instance. I cannot.
Same for releases. 
Maybe the Nexus permissions are too restrictive for anonymous users to substrate?

The snapshots do not appear on MavenCentral, so I need to add oss.sonatype.org.
But I have added it (oss.sonatype.org) as a separate entry.

Reply:
Substrate snapshots are published to Maven Central. Attach and Client plugin snapshots are published to Gluon Nexus. This can be verified [here](https://nexus.gluonhq.com/nexus/content/repositories/public-snapshots/com/gluonhq/).

Reply:
@abhinayagarwal https://repo1.maven.org/maven2/com/gluonhq/substrate/ shows only released versions.

My cursory browsing of interwebs tells me that https://oss.sonatype.org/content/repositories/snapshots is the URL to use for Maven Central snapshots.
And I do find substrate there.
But this repo URL is not included by the MavenCentral() method in Gradle, so it needs to be added separately.

Reply:
Yup, you are right. This change is valid.

My previous comment was for @jperedadnr